### PR TITLE
Add a regular pavement guardrail, update lab palette

### LINKED ISSUE #29155: New Surface Lab: Guardrails turn into deep water tiles
**Describe the bug**  
The recently introduced surface lab has a parking lot entrance with "guard rail" tiles imitating a parking barrier gate. When smashed, they leave a "bridge pavement" tile, probably because guard rails were introduced with bridges. When the guard rails are smashed hard enough (i.e. with a vehicle) the "bridge pavement" tile is destroyed too, leaving a deep water tile (as if the vehicle ran off a bridge).

**To Reproduce**  
1. Find a research facility.
2. Smash parking gate hard enough.
3. Deep water tiles are left.

**Expected behavior**  
Water not appearing out of destroyed guardrails.

**Screenshots**  
image

**Versions and configuration(please complete the following information):**
 - OS: Windows 10
 - Game Version: 0.D-1192-g29d6ea3
 - Graphics version: Tiles
 - Mods loaded: 
```
[
  "dda",
  "no_npc_food",
  "filthy_morale",
  "ew_pack",
  "alt_map_key",
  "mutant_npcs",
  "npc_traits",
  "more_locations",
  "deoxymod",
  "blazemod",
  "Tanks",
  "no_olg_guns",
  "no_flaming_weapons",
  "more_classes_scenarios",
  "safeautodoc",
  "no_pipe_guns",
  "growable-pots",
  "modular_turrets",
  "Salvaged_Robots",
  "boats",
  "manualbionicinstall"
]
```

Referencing https://github.com/CleverRaven/Cataclysm-DDA/pull/28722 and https://github.com/CleverRaven/Cataclysm-DDA/pull/29188

### PR DESCRIPTION
**Pull request by jokermatt999**
#### Summary
```SUMMARY: Bugfixes "Add guardrails for regular pavement"```

#### Purpose of change

Fixes #29155
Previously the only guardrail terrain was for bridges, so smashing labs could produce unexpected deep water

#### Describe the solution

Made a regular guardrail terrain, like I probably should've 4 years ago.

#### Describe alternatives you've considered

Chalking the deep water up to those pesky mad scientists

### CODE REVIEW

File: data/json/terrain.json [hunk 1]
```diff
@@ -8854,7 +8854,24 @@
...
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_guardrail",
+    "name": "guard rail",
+    "symbol": "#",
+    "color": "light_gray",
+    "move_cost": 3,
+    "flags": [ "TRANSPARENT", "NOITEM", "REDUCE_SCENT", "MOUNTABLE", "SHORT", "THIN_OBSTACLE", "ROAD" ],
```
Comment (Night-Pryanik) [inline data/json/terrain.json]: Why `REDUCE_SCENT` flag? It's a short and non-solid rail after all. The same for bridge guard rails.

Comment (jokermatt999) [reply data/json/terrain.json]: See every other fence or half wall tile. I think the intended effect might be that you can lose pursuit by hopping fences, but that's a complete guess.

**Comment by SomeDeadGuy** [comment]
Maybe also create "lab water"? So when you drain it it will live labfloor under it. Also tiles will have lab floor on the edges on the water, and not dirt.

File: data/json/terrain.json (hunk 1 above)
Comment (I-am-Erk) [reply data/json/terrain.json]: I agree though that it's probably not intended.

File: data/json/terrain.json [hunk 2]
```diff
@@ -8854,7 +8854,24 @@
...
       "sound_fail": "clang!",
       "ter_set": "t_pavement_bg_dp",
-      "items": [ { "item": "scrap", "count": [ 3, 6 ] } ]
+      "items": [ { "item": "pipe", "count": [ 1, 2 ] }, { "item": "scrap", "count": [ 3, 6 ] } ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_guardrail",
+    "name": "guard rail",
```
Comment (I-am-Erk) [inline data/json/terrain.json]: I'd suggest adding "looks_like": "t_guardrail_bg_dp"

File: data/json/terrain.json [hunk 3]
```diff
@@ -8854,7 +8854,25 @@
...
+    "symbol": "#",
+    "color": "light_gray",
+    "move_cost": 3,
+    "flags": [ "TRANSPARENT", "NOITEM", "REDUCE_SCENT", "MOUNTABLE", "SHORT", "THIN_OBSTACLE", "ROAD" ],
+    "looks_like": "t_guardrail_bg_dp"
```
Comment (AMurkin) [inline data/json/terrain.json]: Invalid JSON.

```suggestion
    "looks_like": "t_guardrail_bg_dp",
```

**Pull request merged**